### DISCUSSION TITLE
Sheet and React navigation

### DISCUSSION THREAD

**Discussion by varand-pez** [tamagui/tamagui#748] [Q&A]
Hello,

I'd like to know if it's possible to open a sheet when clicking on react navigator header left/right buttons?
I understand that this might not be the best place to ask, but any help would be greatly appreciated.

I'd like to design a login page which is a sheet from tamagui and display it when user clicks on header button.

Thanks.

**Comment by tylerjbainbridge** [comment]
@varand-pez Did you find a solution here?
